starter/pom.xml

```diff
@@ -166,41 +155,6 @@
     </build>
 
     <profiles>
```

Comment:
```suggestion
    <profiles>
        <profile>	
            <!-- Vaadin Production mode is activated using -Pproduction -->	
            <id>production</id>	
            <activation>	
                <activeByDefault>true</activeByDefault>	
            </activation>	
            <properties>	
                <vaadin.productionMode>true</vaadin.productionMode>	
            </properties>	
             <dependencies>	
                <dependency>	
                    <groupId>com.vaadin</groupId>	
                    <artifactId>flow-server-production-mode</artifactId>	
                </dependency>	
            </dependencies>
        </profile>
```

Reply:
I'm pretty sure it doesn't work on OSGi.
`flow-server-production-mode` contains only one resource `web-fragment.xml`  and I don't see any way how it may work in OSGi.

In WAR: the fragment is a part of WAR file , so it's clear which application to apply the fragment to.

In OSGi this jar "will be loaded" as  a standalone bundle and there is no any way to associate the web app bundle with the fragment.

In fact "will be loaded" is false. `flow-server-production-mode` has no OSGi compatible manifest and it won't be deployed at all.
So this config has no any effect in OSGi for many various reasons.
production mode can be enabled via `VaadinMode` annotation.
I will add the parameter to it.

Reply:
Right ... I think it will be fine then

Reply:
Added `productionMode` parameter to the `VaadinMode` annotation.